app/views/notifications/shared/_comment_box.html.erb

```diff
@@ -40,23 +40,24 @@
             data-reactable-id="<%= json_data["comment"]["id"] %>"
             data-category="like"
             data-reactable-type="Comment"
+            aria-label="<%= t("views.reactions.like.aria_label") %>"
```

Comment:
The fixes throughout here are to make sure we have a clear accessible name. Previously the accessible name was being constructed via SVG titles which are unreliable across screen readers, and didn't make a lot of sense (e.g. this changes "heart" to "Like" which is more consistent with our terminology elsewhere)